src/receive_data.py

```diff
@@ -23,10 +23,13 @@ def receive_data(what_data):
   if what_data == "G":
     data["thread_label"] = data["label"]
     data["thread_id"] = data["_id"]
+    data["_id"] = data["_id"].astype(str) + "_____" + \
+                              data["created_at"].astype(str)
 
   training = data.loc[data["training"] == 1]
   unlabeled = data.loc[data["training"] == 0].drop(["label"], axis = 1)
-  return [training, unlabeled]
+  #return [training, unlabeled]
+  return [data, data]
```

Comment:
What's going on here? Why make training and unlabeled data if you're not going to use it?

Reply:
This is when Naveen would run the trained classifier on some unlabeled data, but we only pass in data we have labels for.. I can make it `return [training, _]`